packages/ui/src/stories/Button/button.style.ts

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+import Colors from "../../Colors";
+
+const ButtonStyles = styled.div`
+  button:disabled,
+  button:disabled .component-child {
+    cursor: not-allowed !important;
+  }
+  button {
+    font-size: 1.6rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    cursor: pointer;
+    font-weight: 400;
+    line-height: 1.9rem;
+    transition: background-color ease 0.2s;
+    height: 50px;
+    min-width: 200px;
+    box-shadow: rgba(0, 0, 0, 0.12) 0px 5px 10px 0px;
+    padding: 0;
+    border-radius: 5px;
+    text-decoration: none;
+  }
+  .primary {
+    color: ${Colors.white};
+    background-color: ${Colors.primary};
+    border: 0;
+    i {
+      font-size: 1.5em;
+    }
+    &:hover {
+      top: -4px;
+      box-shadow: 0 30px 60px rgba(0, 0, 0, 0.12);
+      background-color: ${Colors.active};
+    }
+    &:active {
+      background-color: ${Colors.active};
+    }
+    &:disabled {
+      box-shadow: rgba(0, 0, 0, 0.12) 0px 5px 10px 0px;
+      background-color: ${Colors.divider};
+    }
+  }
+  .secondary {
+    box-sizing: border-box;
+    color: ${Colors.white};
+    border: 1px solid ${Colors.primary};
+    background-color: ${Colors.primary};
+    i {
+      font-size: 1.4em;
+    }
+    &:hover,
+    &:active {
+      color: ${Colors.primary};
+      background-color: ${Colors.white};
+      border: 1px solid ${Colors.primary};
+    }
+    &:disabled {
+      color: ${Colors.white};
+      background-color: ${Colors.secondary};
+      border: 1px solid ${Colors.secondary};
+    }
+  }
+`;
+
+export default ButtonStyles;
```

Comment:
Lets not export these as default. In larger project I find it problematic, because on the import any developer can use any name they would like to. In this case this wont be an issue since these are styles and are imported only once but lets for rule of thumb always export consts.

Reply:
I understand, I will change it. We will have to configure our linter rules once we add eslint-airbnb because once you added they advice to use `default export` in files with just one export declaration.

Reply:
Yep right. Some rules we will change. But also we want to keep the changed rules to a minimum.